src/snowflake/cli/plugins/stage/manager.py

```diff
@@ -185,10 +205,12 @@ def execute(
         on_error: OnErrorType,
         variables: Optional[List[str]] = None,
     ):
-        stage_path = self.get_standard_stage_prefix(stage_path)
-        all_files_list = self._get_files_list_from_stage(stage_path)
+        stage_path_with_prefix = self.get_standard_stage_prefix(stage_path)
+        stage_path_parts = self._split_stage_path(stage_path_with_prefix)
+        all_files_list = self._get_files_list_from_stage(stage_path_parts)
+
```

Comment:
These all methods for getting stage path variants could be properties of` StagePathParts`. This would probably imply renaming `StagePathParts` to `StagePath` or something similiar.

WDYT?

Reply:
Good idea, will do